pkg/controllers/terminating/v1alpha1/terminate.go

```diff
@@ -38,110 +37,72 @@ type Terminator struct {
 	coreV1Client  corev1.CoreV1Interface
 }
 
-// cordonNodes takes in a list of expired nodes as input and cordons them
-func (t *Terminator) cordonNodes(ctx context.Context) error {
-	// 1. Get terminable nodes
-	nodeList, err := t.getLabeledNodes(ctx, provisioning.ProvisionerTerminablePhase)
-	if err != nil {
-		return err
+// cordonNode takes in a list of expired nodes as input and cordons them
+func (t *Terminator) cordonNode(ctx context.Context, node *v1.Node) error {
```

Comment:
Just musing, can we name this folder the "termination controller" instead of the "terminating controller".

Reply:
I named it terminating per @JacobGabrielson's comment in the last PR. The other package is named `provisioning` so I believe it's proper here. Do you feel strongly otherwise?

Reply:
Ah I didn't notice this parity.

Reply:
In that case, can we make it termination.go (since it's the "Termination Controller")

Reply:
Sure.